tests/test_web/test_rabbitmq.py

```diff
@@ -0,0 +1,115 @@
+from unittest import mock
+
+import pytest
+from kombu.exceptions import OperationalError
+
+
+def assert_testing(rv, mock_smfsc):
+    response_json = {'error': "Scheduling of request id 2 failed."}
+    assert rv.status_code == 500
+    assert response_json == rv.json
+    assert mock_smfsc.call_count == 2
+
+
+@mock.patch('iib.web.api_v1.handle_add_request')
+@mock.patch('iib.web.api_v1.messaging.send_message_for_state_change')
+def test_catch_add_bundle_failure(mock_smfsc, mock_har, db, auth_env, client):
```

Comment:
Bonus points if you query the database to see if the request is in the failed state.